package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "private": true,
   "scripts": {
-    "build": "yarn verify:node && yarn workspaces foreach -ptvi run build",
+    "decap:build-config": "node scripts/mergeDecapConfig.mjs",
+    "build": "yarn verify:node && yarn decap:build-config && yarn workspaces foreach -ptvi run build",
```

Comment:
I added a new script that gets run when we run yarn build in the root directory. Is there a better place to run this?